maximbircu/devtools-library
# Add ability to set dev tools state programmatically

Closes: 


### What has been done
1 Add `DevTools#updateFromParams` which allows the library consumer to update the tools from a set of params(key-value pairs)
1 Add `DevTools#updateFromBundle` which allows the Android library consumer to update the tools from an Android bundle
1 Made each `DevTools` instance use a different storage key so that the similar tools don't collide

### How to test
1.  Open the sample app passing `--ez toggle-tool true --ez tools-group.toggle-tool true ` as startup arguments;
2. Make sure the changes are applied.
